dask_jobqueue/tests/test_sge.py

```diff
@@ -66,3 +70,70 @@ def test_config_name_sge_takes_custom_config():
     with dask.config.set({'jobqueue.sge-config-name': conf}):
         with SGECluster(config_name='sge-config-name') as cluster:
             assert cluster.name == 'myname'
+
+
+@pytest.mark.env("sge")  # noqa: F811
+def test_taskarrays(loop):  # noqa: F811
+
+    def scale_cluster(cluster, size):
+        """ Starts a small cluster and returns information on jobs and tasks """
+        cluster.scale(size)
+        # waiting to allow cluster to start (no pending jobs)
+        sleep(2)
```

Comment:
is this `sleep(2)` mandatory?

Reply:
I need to check if it can be removed.
I added it to give time for the `cluster.pending_jobs` to be populated.
I was thinking that without it, `pending_jobs` could be empty because the jobs haven't been scheduled, not because they are running.

Do you have a better suggestion?